@xen-orchestra/backups/.USAGE.md

```diff
@@ -0,0 +1,12 @@
+## Changing the default backup configuration
+
+It is possible to change the default backup configuration by editing the XO configuration file (`config.toml`).
+
+There are multiple sections concerning backups:
+
+- `[backups]` contains values related to the general backup configuration
+- `[backups.defaultSettings]`, `[backups.vm.defaultSettings]` and `[backups.metadata.defaultSettings]` contain values related to backup jobs default settings:
+  - `[backups.vm.defaultSettings]` is related to VM backup jobs
+  - `[backups.metadata.defaultSettings]` is related to metadata backup jobs
+  - `[backups.defaultSettings]` is related to all backup jobs
+  - in case of duplicate value, `[backups.vm.defaultSettings]` and `[backups.metadata.defaultSettings]` will prevail over `[backups.defaultSettings]`, and a defined job setting will prevail over any `defaultSettings`
```

Comment:
I think we should like to various part of the code to make the available settings discoverable.

Reply:
IMHO , this will be even harder to maintain and discover , since the changes will need to  be maintained in multiple parts

Reply:
I meant to something more or less stable like https://github.com/vatesfr/xen-orchestra/blob/568e14f7eec7c3ddf685d324ad4733a3ef577995/%40xen-orchestra/backups/_runners/_Abstract.mjs#L7

But I agree there are no perfect answers here.